### DISCUSSION TITLE
Component testing when Angular source is in different directory

### DISCUSSION THREAD

**Discussion by aurelijusrozenas** [cypress-io/cypress#25196] [Component Testing]
Hi,

I am trying to setup Component testing with Angular. Our application has multiple `node_modules` installations:
- `/package.json` - at project root are some dependencies, Cypress is among them.
- `/angular/package.json` - angular source and it's dependencies.
- `/cypress` - cypress configuration and tests.

Is it possible to configure Cypress to use `/angular` directory for source and dependecies (e.g., `@angular/cli` etc)?
The only other solution seems to be to move Cypress installation to `/angular`.

**Comment by lmiller1990** [comment]
It definitely should be possible. Nx does this - they have a separate project for tests.

This PR landed last week: https://github.com/cypress-io/cypress/pull/25090. It might be relevant?

What error are you running into exactly? Can you share a minimal reproduction?
